pkg/server/storage/mysql_test.go

```diff
@@ -37,6 +37,7 @@ func TestMysql(t *testing.T) {
 		Type:     "mysql",
 		Schema:   "sharkey_test",
 		Username: "root",
+		Password: "root",
```

Comment:
There is no good way to set up mysql with an empty root password, it seems, so we need the tests to use a password.

Reply:
seems fine, but I wonder if that breaks local testing?

Reply:
By default, mysql has an empty password. To set it you can do:

```
ALTER USER 'root'@'localhost' IDENTIFIED BY 'root';
```

And then local tests work:

```
$ mysql -e 'CREATE DATABASE sharkey_test;' -uroot -proot
$ go test -v ./...
```